notify/telegram/telegram.go

```diff
@@ -80,6 +80,9 @@ func (n *Notifier) Notify(ctx context.Context, alert ...*types.Alert) (bool, err
 	}
 
 	messageText, truncated := notify.TruncateInRunes(tmpl(n.conf.Message), maxMessageLenRunes)
+	if err != nil {
+		return true, err
```

Comment:
I think this should be `false`, no? The message cannot be retried as it will fail again for the same reason?

Reply:
It will most probably fail again indeed.

However, I see that below in the code `true` is returned if `getBotToken()` returns an error. As far as I understand, `getBotToken()` will not probably recover on its own (only if someone reloads the config or changes the `bot_token_file` on the filesystem), similarly to the template rendering. This is why I chose `true` in this case, but I can change this to `false` if fits better here.

Reply:
I think returning `true` for `getBotToken()` is also a bug 👍

Reply:
Ok, changed this case to `false`